packages/wallets/src/features/cfd/modals/CreatePasswordModal/CreatePasswordModal.tsx

```diff
@@ -11,46 +11,46 @@ const CreatePasswordModal: FC<ComponentProps<typeof CreatePassword>> = ({
     password,
     platform,
 }) => {
-    const { isMobile } = useDevice();
-    if (isMobile) {
+    const { isDesktop } = useDevice();
+    if (isDesktop) {
         return (
-            <ModalStepWrapper
-                renderFooter={() => {
-                    return (
-                        <WalletButton
-                            disabled={!password || isLoading}
-                            isFullWidth
-                            isLoading={isLoading}
-                            onClick={onPrimaryClick}
-                            size={isMobile ? 'lg' : 'md'}
-                        >
-                            {`Create ${PlatformDetails[platform].title} password`}
-                        </WalletButton>
-                    );
-                }}
-                title={''}
-            >
+            <ModalWrapper>
                 <CreatePassword
                     isLoading={isLoading}
                     onPasswordChange={onPasswordChange}
                     onPrimaryClick={onPrimaryClick}
                     password={password}
                     platform={platform}
                 />
-            </ModalStepWrapper>
+            </ModalWrapper>
         );
     }
 
     return (
-        <ModalWrapper>
+        <ModalStepWrapper
+            renderFooter={() => {
+                return (
+                    <WalletButton
+                        disabled={!password || isLoading}
+                        isFullWidth
+                        isLoading={isLoading}
+                        onClick={onPrimaryClick}
+                        size={isDesktop ? 'md' : 'lg'}
+                    >
+                        {`Create ${PlatformDetails[platform].title} password`}
+                    </WalletButton>
```

Comment:
Since we're in the process of migrating from WalletButton to deriv-com/ui Button shouldn't new buttons added just be added as deriv-com/ui Button 🤔

Reply:
Since the changes in this PR are already a lot maybe it should just be done in a separate PR to not increase the scope as we already have cards to address this Button refactoring

Reply:
@lubega-deriv God bless you 🙏🏼